### PR TITLE
81 add new libraries

### LINKED ISSUE #81: Add new libraries
Add following libraries to pom

-  Instancio
https://www.instancio.org/
https://www.baeldung.com/java-test-data-instancio
- Jacoco
https://www.jacoco.org/jacoco/trunk/index.html
https://www.baeldung.com/jacoco

### PR DESCRIPTION
**Pull request by Adrian-Rochminski**
closes: #81

### CODE REVIEW

**Review by Krizzuu — DISMISSED** [review]
seems ok

File: pom.xml [hunk 1]
```diff
@@ -104,6 +104,17 @@
             <version>0.11.5</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <version>0.8.10</version>
```
Comment (HerbMat) [inline pom.xml]: Please move version to properties and use it as variable . It should be done as mapstruct ```mapstruct.version```

File: pom.xml [hunk 2]
```diff
@@ -104,6 +104,17 @@
             <version>0.11.5</version>
```
Comment (HerbMat) [inline pom.xml]: Please fix this as well

**Review by HerbMat — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (HerbMat) [reply pom.xml]: Still not fixed

**Review by HerbMat — CHANGES_REQUESTED** [review]

**Review by HerbMat — APPROVED** [review]

**Pull request merged**